.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-flow && lint-staged
+flow && lint-staged
```

Comment:
It's still detecting a difference because this file is missing a new line at the end. This patch shouldn't contain any changes to `.husky/pre-commit`